matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPatientInMatch.java

```diff
@@ -345,6 +366,22 @@ public AccessLevel getAccess()
         return this.access;
     }
 
+    @Override
+    public AccessType getAccessType()
+    {
+        EntityAccess entityAccess = PERMISSIONS_MANAGER.getEntityAccess(this.patient);
+        AccessLevel useAccess = entityAccess.getAccessLevel();
+        Visibility visibility = entityAccess.getVisibility();
+        if (useAccess.compareTo(MATCH) < 0 && visibility.compareTo(MATCHABLE_VISIBILITY) >= 0) {
+            // matches which are not matchable are filtered out elsewhere. But for MME requests which are
+            // done as a guest user (who has no access to all patients), need to make sure the match can
+            // be processed and returned, for which there should be at least "match" access level
+            useAccess = MATCH;
+        }
+
+        return new DefaultAccessType(this.access, VIEW, MATCH);
```

Comment:
the local `useAccess` variable is never used? It is set but then the function does not use it anywhere and returns something which does nto depend on it

Reply:
P.S. I wonder, do we still really need access stuff? We show matched genes either way, do we need it to see the variants?